spec/fixtures/ial2_test_credential_multiple_doc_auth_failures_front_side_only.yml

```
@@ -0,0 +1,6 @@
+doc_auth_result: Failed # values: Passed, Failed, Attention, Unknown
```

```suggestion
transaction_status: failed
doc_auth_result: Failed # values: Passed, Failed, Attention, Unknown
```
for incoming [LG-13005](https://cm-jira.usa.gov/browse/LG-13005) 😄